src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -303,13 +306,20 @@ public List<JobModel> enumerateJobs(
       filter.addFilterFlightClass(FlightFilterOp.EQUAL, className);
     }
 
+    HashSet<String> jobIdSet = new HashSet<>(ListUtils.emptyIfNull(jobIds));
```

Comment:
Also, instead of handling `null` I would advocate for making this API require that the caller always pass a value. The only caller is in the controller layer, so you could check for `null` there. I think a good general rule is that collection parameters (and results) should never be null, as long it's in code we control.